platformio.ini

```diff
@@ -35,10 +35,10 @@ custom_unpack_dir           = unpacked_littlefs
 build_unflags               = ${core.build_unflags}
 build_flags                 = ${core.build_flags}
 monitor_speed               = 115200
-monitor_port                = COM5
+monitor_port                = /dev/ttyUSB0

```

Comment:
Do not change to /dev/ttyUSB0

Reply:
See https://github.com/arendst/Tasmota/pull/16014